meta-linaro-mbl/meta-optee/recipes-security/optee/optee-os_%.bbappend

```diff
@@ -25,6 +25,8 @@ OPTEEMACHINE_imx8mmevk-mbl = "imx-imx8mmevk"
 OPTEEOUTPUTMACHINE_imx="imx"
 OPTEEMACHINE_raspberrypi3-mbl="rpi3"
 OPTEEOUTPUTMACHINE_raspberrypi3-mbl="rpi3"
+OPTEEMACHINE_beaglebone-mbl="ti-am43xx"
```

Comment:
We are not using optee, right? Because CPU on BB is am335x, not am43xx. If so, you shall add comments to mark that it is temporary dummy solution and we should change machine name after am335x is ready.

Reply:
OK. Thanks. I've added the comments. Please review it again. Thanks.